packages/gradle/src/plugin/nodes.ts

```diff
@@ -181,9 +182,9 @@ function createGradleTargets(
 
     const outputs = outputDirs.get(task.name);
     targets[targetName] = {
-      command: `${getGradleBinaryPath()} ${task.name}`,
+      command: `${getGradleFile()} ${task.name}`,
       options: {
-        cwd: projectRoot,
+        cwd: workspaceRoot,
```

Comment:
Can we keep `projectRoot` and get a relative path from `projectRoot` to `gradleFile`?

Moving to the `workspaceRoot` would break the command for nested projects.